tests/eval_tests/tests.py

```diff
@@ -251,6 +327,42 @@ def main():
     # Get server configurations
     server_configs = get_server_configs()
 
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--functions",
+        type=str,
+        required=False,
+        help="Client tool test variation",
+    )
+
+    args = parser.parse_args()
+
+    client_tool_module_map = {
+        "tools/tools.py": tools,
+        "tools/tools_bad_function_names.py": tools_bad_function_names,
+        "tools/tools_no_extra_tags.py": tools_no_extra_tags,
+        "tools/tools_only_params.py": tools_only_params
+    }
+
+    selected_client_tool_module = None
+    if args.functions:
+        if args.functions in client_tool_module_map:
+            selected_client_tool_module = client_tool_module_map[args.functions]
+            logger.info(f"Successfully imported functions from {args.functions}")
+        else:
+            logger.error(f"Invalid function name: {args.functions}")
+            return
+    else:
+        logger.info("No --functions argument provided. Using tools/tools.py as default.")
+        selected_client_tool_module = tools
+
```

Comment:
If you add the default value from the comment above, then I don't think you need this if/else statement. `if args.functions` will always be true, and if no functions was specified it will default to `tools/tools.py`